### DISCUSSION TITLE
Welcome to beercss Discussions!

### DISCUSSION THREAD

**Discussion by beercss** [beercss/beercss#117] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by laurentpayot** [comment]
Hi there, I’m a web developer (F# + TS) from Paris, France.
My goal is to have native-like UI in PWAs with minimal CSS/JS footprint. Beercss is great for that!
My favorite beer is a [red ale brewed in the french alps](https://www.brasserie-montblanc.com/en/product/rousse-3/) :beers:

**Comment by leonardorafael** [comment]
Hello 👋, I'm from south of Brazil, my favourite beer is weizen beer (after beercss of course 😄). I love frontend dev and pwa capabilities. That's it ,🍻

**Comment by AlTrAugsburg** [comment]
Hi, I'm hobby/professional web developer from Bavaria, Germany. I discovered Beercss while working on a paper for my computer science study a year ago and fallen in love with it and the approach ❤️
My goal is to learn more about frontend design and hopefully create some great applications with Beercss, cheers 🍻

**Comment by KicKerBNU** [comment]
Hello, I´m from Santa Catarina, Brazil. Working as a frontend developer with VueJS, i love VueJS and BeerCss. My favourite beer today is Blond Ale 🍺. My goal here is to learn and help the BeerCss community.

**Comment by bariccattion** [comment]
Hello, I'm from Paraná, Brazil. I work as a Tech Lead, on the Front-end side working with React/Next.
I've discovered BeerCSS when searching for Material 3 Libraries, and found this solution to be the best and smartest, adding minimal CSS/JS and so helping with future maintainability. I hate adding dependencies to a project, and would like to use just CSS/JS as much as possible
I'm more of a Whisky guy, but my favorite beer is India Pale Ale. 
My goal here is to ask questions, give suggestions, and maybe help the beercss community overall.

**Comment by acroyear** [comment]
From the Washington, DC area in the states (the Virginia side). I'm a "malt-forward" type - so Oktoberfests, Amber ales, British bitters (cask engine-pulled, yes there are places in the states that brew them), and properly made Helles and Pilsners) tend to be my goto's.

I write and maintain clients for the Subsonic music server, and am using beercss (plus michijs for web-components) to rewrite my core player app away from react and mui, to a material 3 style. when i started, this was more stable and polished than google's official material web (and it has the dialog/drawer stuff that the official components don't have yet even in their 1.0).

**Reply by acroyear** [reply]
https://github.com/michijs/michijs - a declarative web-component builder with jsx support. works pretty nice (a few hiccups but it looks like his upcoming v2 is going to resolve most).

https://subfiresuite.com/demo5/#/home - the music player, though i don't have it yet working with a 'guest' login at various demo servers. i'm hoping to add that over this upcoming holiday weekend. The player will use the album cover as the theme source and so change the colors appropriately.
